
<launch>
    <!-- VIO through flvis -->
    <!-- <include file="$(find flvis)/launch/flvis_d455_depth.launch"/> -->

    <!-- ROS Nodelet manager -->
    <!-- <node pkg="nodelet" type="nodelet" name="alan_master" args="manager" output="screen">
            <rosparam command="load" file="$(find alan_landing_planning)/launch/configs/alan_pose_estimation.yaml" />
    </node>  -->


    <node name="alan_master_planner_node" pkg="alan_landing_planning" type="planner_node.cpp" output="screen">
            <rosparam command="load" file="$(find alan_landing_planning)/launch/configs/alan_land_trajectory.yaml" />
            <param name="/log_path" type="string" value="$(find alan_landing_planning)/src/log/"/>
    </node>    

</launch>
